src/py42/clients/auditlogs.py

```diff
@@ -0,0 +1,93 @@
+class AuditLogsClient(object):
+    def __init__(self, audit_log_service):
+        self._audit_log_service = audit_log_service
+
+    def get_page(
+        self,
+        page_num=1,
+        page_size=None,
+        begin_time=None,
+        end_time=None,
+        event_types=None,
+        user_ids=None,
+        usernames=None,
+        user_ip_addresses=None,
+        affected_user_ids=None,
+        affected_usernames=None,
+        **kwargs
+    ):
+        """Retrieve a page of audit logs, filtered based on given arguments.
+
+        Note: `page_num` here can be used same way as other methods that have a
+        `page_num` parameter in py42. However, under the hood, it subtracts one from
+        the given `page_num` in the implementation as the Code42 Audit-Logs API expects
+        the start page to be zero.
+
+        Args:
+            page_num (int, optional): The page number to get. Defaults to 1.
+            page_size (int, optional): The number of items per page. Defaults to `py42.settings.items_per_page`.
+            begin_time (int, optional): Timestamp in milliseconds. Defaults to None.
+            end_time (int, optional): Timestamp in milliseconds. Defaults to None.
+            event_types (str or list, optional): A str or list of str of valid event types. Defaults to None.
+            user_ids (str or list, optional): A str or list of str of user ids. Defaults to None.
```

Comment:
Are these the legacy IDs or the user UIDs ?

Reply:
Thanks, changed to user uid.